src/main/java/org/opensearch/commons/ConfigConstants.java

```diff
@@ -21,7 +21,7 @@ public class ConfigConstants {
     public static final String AUTHORIZATION = "Authorization";
 
     // These reside in security plugin.
-    public static final String OPENSEARCH_SECURITY_SSL_HTTP_PEMCERT_FILEPATH = "plugins.security.ssl.http.pemcert_filepath";
+    public static final String OPENSEARCH_SECURITY_SSL_HTTP_PEMTRUSTEDCAS_FILEPATH = "plugins.security.ssl.http.pemtrustedcas_filepath";
```

Comment:
This is going to break a few plugins with this name change. If we go ahead with this, issues to each of the plugins should be made to indicate this change.

Affected plugins: https://github.com/search?q=org%3Aopensearch-project%20%22OPENSEARCH_SECURITY_SSL_HTTP_PEMCERT_FILEPATH%22&type=code

Reply:
Also is it necessary to have a variable name change here?